src/wallet/walletutil.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2017 The Bitcoin Core developers
```

Comment:
Agree on creating a separate walletutil, this avoids picking up #ifdef WALLET_ENABLE logic in util.cpp.